src/_actions/statusMessageToggle.action.js

```diff
@@ -0,0 +1,14 @@
+import constants from '../_constants';
+
+const {
+  TOGGLE_STATUS_MESSAGE_VISIBILITY,
+  CLOSE_STATUS_MESSAGE
+} = constants;
+
+export const toggleMessageVisibility = () => ({
```

Comment:
At first visibility was in state. I wanted to give the user the option of closing the message. And after closing the message they have an option of adding another asset. But with visibility in state, the message would remain closed. I enquired from other devs and we came to the conclusion of saving it in the store. @sachungo

Reply:
@kimotho-njoki not necessarily. If you look at the code example I gave, the message will be shown whenever we have an error or success message. That's why I added the reset method to reset the messages where necessary. Each time the component is mounted, it will be shown unless the user closes it themselves